Userland/Libraries/LibJS/Runtime/ArrayBuffer.cpp

```diff
@@ -21,7 +21,7 @@ ArrayBuffer* ArrayBuffer::create(GlobalObject& global_object, ByteBuffer* buffer
 
 ArrayBuffer::ArrayBuffer(size_t byte_size, Object& prototype)
     : Object(prototype)
-    , m_buffer(ByteBuffer::create_zeroed(byte_size))
+    , m_buffer(ByteBuffer::create_zeroed(byte_size).release_value()) // FIXME: Handle this possible OOM failure.
```

Comment:
Note that the spec does explain how to handle this OOM failure: https://tc39.es/ecma262/#sec-createbytedatablock

Reply:
Noted, I'll see if it's simple enough to move the allocation out of the constructor and handle it there.